README.md

```diff
@@ -98,15 +98,21 @@ If you need to run the backup job outside the schedule, use:
     tutor k8s backup
 
 The backup job backs up MySQL, MongoDB, and the Caddy data directory, 
-creates a tar file, and uploads it to an S3 storage bucket as set by 
-the `BACKUP_S3_*` configuration parameters. Note that the bucket must support 
-[object versioning](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html).
+creates a tar file with a date stamp, and uploads it to an S3 storage bucket as 
+set by the `BACKUP_S3_*` configuration parameters. Note that the bucket must 
+support 
+[object versioning](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html)
+if you want to run multiple backups during the day.
```

Comment:
Technically, it doesn't even _have to_. It's just that if people want to run multiple backups in a day, and they _don't_ enable versioning, then only the last backup taken on any day survives.

Reply:
I'll change the wording there to make it more clear.